R/helpers.R

```diff
@@ -3516,7 +3539,7 @@ default_fonts <- function() {
 #' ## Slab Serif (`"slab-serif"`)
 #'
 #' ```css
-#' font-family: Rockwell, 'Rockwell Nova', 'Roboto Slab', 'DejaVu Serif', 'Sitka Small', serif;
+#' font-family"=ockwell, 'Rockwell Nova', 'Roboto Slab', 'DejaVu Serif', 'Sitka Small', serif;
```

Comment:
This looks like an accidental edit. Could you revert?

Reply:
Ah, yes!